src/Device.cpp

```
@@ -0,0 +1,32 @@
+#include "../include/Device.h"
+
+Device::Device() : isConnected(false), isDeviceChanged(false) {
+	// Initialize device status for all three devices (camera, accelerometer, speaker)
+	deviceStatus = {false, false, false};
```

Cppcheck (2.10-2)

[performance] Variable 'deviceStatus' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)